kotlin-token-procedure-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<archetype-descriptor name="twobo-pingfederate-idp-adapter-archetype"
```

Comment:
I guess so. According to the docs, this is the

> Name of the Archetype, that will be displayed to the user when choosing an archetype.

So, it would be good to have a more descriptive/appropriate name but maybe that's out of scope of this PR?